CodingGuidelines.md

```diff
@@ -0,0 +1,34 @@
+# Coding Guidelines
+
+The coding guidelines in ezEngine are enforced through clang-tidy. You can either run clang-tidy locally or use the automated CI process that runs in every PR into ezEngine. CI will provide a git patch with all suggested changes which you can apply locally. After applying the suggested changes please make sure everything still compiles. The fixes done by clang-tidy are not garantueed to work in all cases.
+
+## Type dependant prefixes 
+
+These type dependant prefix is currently only mandatory for member variables. Function / Method parameter names and variables in Function / Method bodies can be named to the programmers liking.
+
+* If the variable / member is a `ezInt8`, `ezInt16`, `ezInt32`, `ezInt64` or `ptrdiff_t` the prefix is 'i': `ezInt32 iMyVar;`
+ * If the member is a `ezUint8`, `ezUInt16,` `ezUInt32`, `ezUInt64` or `size_t` the prefix is 'ui': `ezUInt32 uiMyVar;`
+ * If the member is a `float` or `double` the prefix is 'f': `float fMyVar;`
+ * If the member is a `bool` the prefix is 'b': `bool bMyVar;`
+ * If the member is a handle, the prefix is 'h': `ezSpatialDataHandle hMyVar;`
+ * If the member is a raw pointer, `ezSharedPtr` or `ezUniquePtr` the prefix is 'p': `ezUInt32* MyVar;`
+ * If the member is a `const char*` the prefix is 'sz': `const char* szName;`
+ * If the member is a ezEngine string (`ezString`, `ezStringView`, etc) the prefix is 's': `ezString sMyVar;`
+
+## Members of structs and Classes
+
+### Non-Static
+
+If the member is public, no rules apply.
+
+For private and protected members, the following rules apply:
+ * All members must start with 'm_' (this comes before the type dependant prefix)
+ * The name of the static member must be in PascalCase: `s_MyMember` 

```

Comment:
this is the non static member section, thus this sentence and example is weird

Reply:
I think this is copy paste. Just remove the 'static' and change to m_MyMember